.github/workflows/coverage.yml

```diff
@@ -0,0 +1,28 @@
+name:                           coverage
+
+on:                             [push]
+jobs:
+  test:
+    name:                       coverage
```

Comment:
Nit: The spacing/indentation scheme is different than the other yml file in this directory.